docs/community/get_involved.md

```diff
@@ -7,28 +9,76 @@
 
 </div>
 
+## GTFS Governance
+
+The GTFS Specification is not set in stone. Instead, it is an open specification developed and maintained by the community of transit agencies, developers, and other stakeholders who use GTFS. It is expected that this community of producers and consumers of GTFS data will have proposals for extending the spec to enable new capabilities.
```

Comment:
This text is included in each Spec Amendment Process, should we remove it from there and keep it in here?

Reply:
id keep it